backend/de.metas.banking/de.metas.banking.camt53/src/main/java/de/metas/banking/camt53/wrapper/v04/NoBatchBankToCustomerStatementV04Wrapper.java

```diff
@@ -38,16 +38,13 @@
 @Value(staticConstructor = "of")
 public class NoBatchBankToCustomerStatementV04Wrapper
 {
-	private static final AdMessageKey MSG_BATCH_TRANSACTIONS_NOT_SUPPORTED = AdMessageKey.of("de.metas.banking.camt53.wrapper.NoBatchBankToCustomerStatementV02Wrapper.BatchTransactionsNotSupported");
-
 	@NonNull
 	BankToCustomerStatementV04 bankToCustomerStatementV04;
 
 	private NoBatchBankToCustomerStatementV04Wrapper(@NonNull final BankToCustomerStatementV04 bankToCustomerStatementV04)
 	{
 		bankToCustomerStatementV04
```

Comment:
can'T we drop this and the following line alltogether? and then maybe rename this class as well? (removing the `No`)